Document the new notifications (#880)
neo4j/docs-cypher: modules/ROOT/pages/deprecations-additions-removals-compatibility.adoc

[[cypher-deprecations-additions-removals-5.17]]
== Neo4j 5.17

=== Updated features

[cols="2", options="header"]
|===
| Feature
| Details

a|
label:functionality[]
label:updated[]
[source, cypher, role=noheader]
----
CREATE [index_type] INDEX [index_name] IF NOT EXISTS FOR ...
----
|
When attempting to create an index using `IF NOT EXISTS` with either the same name or same index type and schema, or both,

> as an existing index the command now returns a notification showing the existing index which blocks the creation.

Reviewer: Please don't break text in-between sentences. Either break it after full stops, or not at all. There's multiple occurrences in this file. After addressing this, it's good to merge as far as I can see :cartwheeling:

Author: I did break them while writing to be able to see the sentences since they were so long (as I don't have soft wrapping so the lines just continue out of sight and requires scrolling to see the end) but I can definitely go back and fix that 👍

> as an existing index the command will now return a notification showing the pre-existing index blocking the creation.

Reviewer 2: ```suggestion
as an existing index the command now returns a notification showing the existing index which blocks the creation.
```

a|
label:functionality[]
label:updated[]
[source, cypher, role=noheader]
----
CREATE CONSTRAINT [constraint_name] IF NOT EXISTS FOR ...
----
|
When attempting to create a constraint using `IF NOT EXISTS` with either the same name or same constraint type and schema

> (and property type for property type constraints), or both, as an existing constraint the command will now return
> a notification showing the pre-existing constraint blocking the creation.

Reviewer 2: ```suggestion
(and property type for property type constraints), or both, as an existing constraint the command now returns
a notification showing the existing constraint which blocks the creation.
```
